src/trial_synth/config_environ.py

```diff
@@ -0,0 +1,119 @@
+import os
+import sys
+import shutil
+import logging
+if sys.version_info[0] == 3:
+    from configparser import RawConfigParser
+else:
+    from ConfigParser import RawConfigParser
+
+logger = logging.getLogger(__name__)
+
+# If the configuration file does not exist, try to create it from the default
+home_dir = os.path.expanduser('~')
+config_dir = os.path.join(home_dir, '.config', 'who-ictrp')
+config_path = os.path.join(config_dir, 'config.ini')
+default_config_path = os.path.join(os.path.dirname(__file__),
+                                   'resources/default_config.ini')
+if not os.path.isfile(config_path):
+    try:
+        os.makedirs(config_dir)
+    except Exception:
+        logger.warning(config_dir + ' already exists')
+    try:
+        shutil.copyfile(default_config_path, config_path)
+    except Exception:
+        logger.warning('Could not copy default config file.')
+
+
+def populate_config_dict(config_path):
+    """Load the configuration file into the config_file dictionary
+
+    A ConfigParser-style configuration file can have multiple sections, but
+    we ignore the section distinction  and load the key/value pairs from all
+    sections into a single key/value list.
+    """
+    try:
+        config_dict = {}
+        parser = RawConfigParser()
+        parser.optionxform = lambda x: x
+        parser.read(config_path)
+        sections = parser.sections()
+        for section in sections:
+            options = parser.options(section)
+            for option in options:
+                config_dict[option] = str(parser.get(section, option))
```

Comment:
One issue here is that we are adding `option` to `config_dict` and `section` gets ignored. This means that keys have to be unique across sections in the config file. Is that the intention?